lib/createJestRunner.ts

```diff
@@ -87,9 +89,7 @@ export default function createRunner<
       onFailure: OnTestFailure,
       options: TestRunnerOptions,
     ): Promise<void> {
-      const runner: (runTestOptions: RunTestOptions) => TestResult = (
-        await import(runPath)
-      ).default;
+      const runner: TestRunner = (await import(runPath)).default;
```

Comment:
Do I get it right, `runPath` now can be `string | URL`?

Reply:
it's just `string`. We could accept either tho

I don't think `jest-worker` accepts a URL, we could fix that and simplify a tiny bit here

Reply:
`jest-worker` take only strings. I was thinking to add URL support after making it async, but got stuck with the async PR https://github.com/facebook/jest/pull/12680#discussion_r945123483

Reply:
`jets-worker` now handles taking a `URL`